src/App.jsx

```diff
@@ -1,5 +1,54 @@
+/** @jsxImportSource @emotion/react */
+import { css } from '@emotion/react';
+import Header from './Components/Header';
+import { useEffect, useState } from 'react';
+import { getLocalStorage } from './utils/getLocalStorage';
+import List from './Components/List';
+
 function App() {
-  return <div>React Todo</div>;
+  const [todoList, setTodoList] = useState(getLocalStorage() || []);
+  useEffect(() => {
+    if (getLocalStorage() === null) {
+      localStorage.setItem('todo', JSON.stringify([]));
+    }
+  }, []);
+  const notDoneTodoList = todoList.filter((todo) => todo.done === false);
+  const doneTodoList = todoList.filter((todo) => todo.done === true);
+  return (
+    <div
+      css={css({
+        width: '100vw',
+        height: '100vh',
+        color: 'black',
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+        background: 'linear-gradient(pink,skyblue)',
+      })}
+    >
+      <div
+        css={css({
+          width: '360px',
+          height: '600px',
+          backgroundColor: 'white',
+          borderRadius: '20px',
+          boxShadow: '0px 0px 25px rgba(0, 0, 0, 0.25)',
+        })}
+      >
+        <Header setTodoList={setTodoList} />
+        <List
+          listName="📋 TO DO"
```

Comment:
이런 매직 스트링은 지양하는 것이 좋습니다. 상수로 분리해주세요.

```suggestion
const LISTNAME = {
    TODO: "TO DO",
    DONE: "DONE",
}

<List
    listName={`📋 ${LISTNAME.TODO}`}
```